src/style.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+.smallSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+.first_form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.second_form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.bigSearchBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+}
+
+.smallSearchBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  padding-bottom: 2px;
+  height: 42px;
+  padding-left: 33px;
+}
```

Comment:
Same here, just apply big searchbar class to small and override sizes styles for small search bar, don't repeat yourself